packages/components/src/QualityBar/QualityBar.component.js

```diff
@@ -5,6 +5,7 @@ import RatioBar from '../RatioBar';
 import {
 	QualityEmptyLine,
 	QualityInvalidLine,
+	QualityType,
```

Comment:
don't forget to export it on the component so it can be used in UMD/CDN mode
in `packages/components/src/QualityBar/index.js`:
add `QualityBar.QualityType = QualityType` (you can look at CircularProgress component for example)